src/home_robot/home_robot/mapping/voxel.py

```diff
@@ -312,6 +365,17 @@ def write_to_pickle(self, filename: str):
         with open(filename, "wb") as f:
             pickle.dump(data, f)
 
+    def fix_data_type(self, tensor) -> torch.Tensor:
```

Comment:
I like this method. Small nit: I'd probably just call this `ensure_float_tensor`